### DISCUSSION TITLE
Transitions

### DISCUSSION THREAD

**Discussion by pike2k** [olikraus/u8g2#1653] [Q&A] [answered]
Still learning u8g2 here. I wonder if there's any suppport for transitions like 'ESP8266 Oled Display Library' has?

**Comment by olikraus** [comment]
What exactly do you mean by ''transitions'?

**Comment by pike2k** [comment]
Here's some info https://github.com/ThingPulse/esp8266-oled-ssd1306#ui-library-oleddisplayui

In the introduction to this video the animation/transition can be seen: https://www.youtube.com/watch?v=McwarmDpQXo
The code used in the above video is located here: https://github.com/ThingPulse/esp8266-weather-station/tree/master/examples/WeatherStationDemo

**Comment by olikraus** [comment]
This isn't implemented in u8g2, however it could be implemented easily with u8g2. The u8g2 graphics demo demonstrates transitions as an example.

**Reply by pike2k** [reply]
I will analyze your code-example. Hopefully I can make sense of it. Thanks

**Comment by pike2k** [comment, marked as answer]
I've now looked at your code example "Graphics demo". 
Indeed it contains animations but no transitions? If you look at the youtube link above again, the transitions starts at approx 13 seconds and shows when part of the screens content is replaced with other content in scrolling motion. Since only part of the content is replaced, I guess this is part of the UI mode I also linked in an earlier comment.

**Reply by olikraus** [reply]
U8g2 is a lib, which provides access to a graphics device. The content needs to defined by you.

**Reply by pike2k** [reply]
Yes I understand that now :) I think I already knew it, I just wanted to make sure that I didn't miss any features that was not known to me.
